js/binance.js

```diff
@@ -1913,7 +1919,13 @@ module.exports = class binance extends Exchange {
         } else if ((type === 'margin') || (marginMode === 'cross')) {
             method = 'sapiGetMarginAccount';
         } else if (type === 'savings') {
-            method = 'sapiGetLendingUnionAccount';
+            const lockedProduct = this.safeStringUpper (params, 'lockedProduct');
+            if (lockedProduct !== undefined) {
+                request['product'] = lockedProduct;
```

Comment:
@sc0Vu in this case, since we're doing nothing with `product` we don't need to extract it from `params` and add it to `request`, because in the end `request` is extended with `params` .